### DISCUSSION TITLE
How to detect if video and audio feed of a stream are active

### DISCUSSION THREAD

**Discussion by vladimir0802** [ant-media/Ant-Media-Server#6391] [FAQs]
Hi,

My team is facing a problem with recognizing if there are **active** video and audio feed of a stream or not. What is actual problem is when user (stream owner) quit the app but did not end the stream properly. That specific stream keeps showing "broadcasting" state in Ant Media Server manager and we are **unable to identify that**. We would like to implement a script that will look for a streams with no active video and audio feed and if that check comes true more than once, the stream should end (will quit it through REST api).

**Comment by amarantmeida** [comment]
@vladimir0802 You can use the below RestAPI to check the stream is active or not. 
https://antmedia.io/rest/#/default/getBroadcast

Parameter updateTime must be a timestamp when stream is last updated. If the difference between "currenttimemilis" and "updateTime"is 20+ seconds pass from current time, we say its not updated -> dead stream. 
Please review and let me know for any follow-up queries.

**Comment by vladimir0802** [comment]
@amarantmeida 

Thanks for the response, updateTime is exactly what we needed. 

Now we are faced with another problem. We cannot stop nor delete stream using REST apis (tried with https://antmedia.io/rest/#/default/deleteBroadcast and https://antmedia.io/rest/#/default/stopStreamingV2 but both of them returns 404). I  guarantee the stream was active when the stop/delete APIs were called...

**Reply by yashtandon113** [reply]
@vladimir0802 

If the API call returns 404 error, then it could be possible that API call is wrong or any other requested parameter is not available. Can you share your curl sample?
